training/galaxy.py

```diff
@@ -128,7 +128,7 @@ class IntentionalRollback(Exception):
             role
         WHERE
             type = 'system'
-            AND name = 'training-' || %s
+            AND name LIKE 'training-%' || %s
```

Comment:
```suggestion
            AND name LIKE 'training-%%' || %s
```

otherwise it'll throw an error when templating